pages/dkp/kaptain/2.0.0/install/air-gapped-2.1/index.md

```diff
@@ -0,0 +1,132 @@
+beta: false
+enterprise: false
+---
+
+<p class="message--warning"><strong>WARNING: </strong>
+You can deploy Kaptain to a cluster in a selected workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the workspace you are deploying to or move that cluster to another workspace.
+</p>
+
+## DKP 2.1 air-gapped installation
+
+Refer to [DKP install instructions][dkp_install], if you want to deploy Kaptain in a networked environment or to [DKP 2.2 air-gapped instructions][2.2_air] if you are deploying in DKP 2.2.
+
+Kaptain supports installation on an air-gapped (a.k.a. offline or private) DKP managed cluster. Before installing Kaptain, please follow the [air-gapped installation guide][konvoy-air-gap] to set up the air-gapped DKP managed cluster. The cluster admin is responsible for configuring the DKP cluster correctly and ensuring container images have been pre-loaded to the private registry before installing Kaptain.
+
+## Prerequisites
+
+-   A DKP cluster with the following Platform applications enabled:
+
+    - Istio
+    - Knative (optional, if KServe is configured to work in `RawDeployment` mode)
+
+-   [`kubectl`][kubectl] on your installation machine
+
+-   For customers deploying in a multi-cluster environment (Enterprise): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex].
+
+-   Ensure the following applications are enabled in Kommander:
+
+    1. Use the existing Kommander configuration file, or initialize the default one:
+
+		```bash
+		dkp install kommander --init > kommander-config.yaml
+		```
+
+    1. Ensure the following applications are enabled in the config:
+
+		```yaml
+		apiVersion: config.kommander.mesosphere.io/v1alpha1
+		kind: Installation
+		apps:
+		  ...
+		  dex:
+		  dex-k8s-authenticator:
+		  kube-prometheus-stack:
+		  istio:
+		  knative:
+		  minio-operator:
+		  traefik:
+		  nvidia:  # to enable GPU support
+		  ...
+		```
+
+    1. Apply the new configuration to Kommander:
+
+		```bash
+		dkp install kommander --installer-config kommander-config.yaml
+		```
+
+<!-- TODO: Reference doc for DKP, as commands are different depending on the installation and delete only this step or the whole "Ensure the following applications are enabled in Kommander:" part?-->
+
+  Review the [Kommander installation documentation][kommander-install] for more information.
+
+<p class="message--note"><strong>NOTE: </strong>Starting from the 1.3 release, Spark Operator is no longer installed by default with Kaptain.</p>
+
+In case you need to run Spark jobs on Kubernetes using Spark Operator, it needs to be installed separately.
+Use the following instructions to install Spark Operator from Kommander Catalog for [DKP 2.x][install-spark-dkp2].
+
+### Load the Docker images into your Docker registry
+
+1.  Download the image bundle file:
+
+    - Download `kaptain_air_gapped.tar` that will contain the required artifacts to perform an air-gapped installation.
```

Comment:
@lukeogg @alembiewski Is this the tarball step you mention required to run the `dkp push chart [chartTarball]`?
```suggestion
    - Download `kaptain_air_gapped.tar` that will contain the required artifacts to perform an air-gapped installation. The kommander-image-bundle.tar tarball has the required artifacts.
```